### DISCUSSION TITLE
Can I use C++20 or C++23 instead of C++17?

### DISCUSSION THREAD

**Discussion by kmc7468** [ANLAB-KAIST/KENSv3#134] [Project Q&A 1. Application Layer (Socket)] [answered]
Can I use C++20 or C++23 instead of C++17?

I edited CMakeLists.txt as follow to use C++20 and I think it works well:
```
16 | set(CMAKE_CXX_STANDARD 20)
```

**Comment by Pusnow** [comment, marked as answer]
No, please use C++17 and below only.

To make KENS backward-compatible, we have decided to stick with C++17.

What C++20 features do you want to use?

**Reply by kmc7468** [reply]
Thank you for your fast reply!

I wanted to use the designated initializer to initialize the `sockaddr_in` structure conveniently.
But I will return to C++17 now.
